e2e/tests/functional/plugins/operatorStatus/operatorStatus.e2e.spec.js

```diff
@@ -63,6 +63,11 @@ test.describe('Operator Status', () => {
     await expect(page.locator('.c-status-poll-panel')).toBeVisible();
   });
 
+  // Description should be empty https://github.com/nasa/openmct/issues/6978
+  test('no description in role selection', async ({ page }) => {
+    await expect(page.locator('.c-message__action-text')).toBeHidden();
```

Comment:
Couple suggestions here.

Instead of creating a whole new test for this, let's just add an assertion to the `beforeEach` since that's where we open the user role dialog. Let's also assert that the text is an empty string.

Reply:
Done! Due the nature of the `v-if` conditional, the section isn't rendered, so it won't be visible 
<img width="896" alt="Screenshot 2023-08-28 at 22 23 32" src="https://github.com/nasa/openmct/assets/16735925/18b1e4dd-456d-4272-999c-8bce406af638">

Reply:
That works!

Reply:
Good stuff. Lmk if you need anything else